Makefile

```diff
@@ -125,12 +125,12 @@ debug-run-nucliadb-redis:
 
 build-node-binding:
 	rm -rf target/wheels/*
-	maturin build -m nucliadb_node_binding/Cargo.toml --profile release-wheel
+	RUSTFLAGS="--cfg tokio_unstable" maturin build -m nucliadb_node_binding/Cargo.toml --profile release-wheel
```

Comment:
I think you should add `--cfg prometheus_metrics` too. @hermeGarcia?

Reply:
I am also curious why we can't have all the options in https://github.com/nuclia/nucliadb/blob/main/.cargo/config.toml

Reply:
`--cfg prometheus_metrics` gives us errors with the bindings

Now, I also understand why the build server was failing. This is annoying.